datafusion/proto-common/src/generated/pbjson.rs

```diff
@@ -3911,6 +3913,7 @@ impl<'de> serde::Deserialize<'de> for JoinType {
                     "RIGHTSEMI" => Ok(JoinType::Rightsemi),
                     "RIGHTANTI" => Ok(JoinType::Rightanti),
                     "LEFTMARK" => Ok(JoinType::Leftmark),
+                    "RIGHTMARK" => Ok(JoinTYpe::Rightmark),
```

Comment:
This is causing CI failures. I belive this file should be generated by running: https://github.com/apache/datafusion/blob/main/datafusion/proto-common/regen.sh
```suggestion
                    "RIGHTMARK" => Ok(JoinType::Rightmark),
```

Reply:
Is this included in the docs?

Reply:
There is some documentation here: https://github.com/eejbyfeldt/datafusion/blob/2e52580b0f10c35b36d02c6f44e13a460de771c9/datafusion/proto/CONTRIBUTING.md?plain=1#L20-L32 But I am sure it could be improved.

Reply:
Yeah I can probably open a pr to add this later.